src/main/java/shop/menu/LoginMenu.java

```diff
@@ -0,0 +1,71 @@
+package shop.menu;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class LoginMenu implements Menu {
+    private Scanner scanner = new Scanner(System.in);
+    private List<String> options = new ArrayList<>();
+    private String name;
+    private String email;
+    private String phoneNumber;
+    private String password;
+    private boolean start;
+    //privet UserService userService = newUserService();
+
+    public void addOptions() {
+        options.add("1. Login");
+        options.add("2. Register user");
+        options.add("0. Exit");
+    }
+
+    public void getLoginAndPassword() {
+        System.out.println("Enter email");
+        email = scanner.nextLine();
+        System.out.println("Enter password");
+        password = scanner.nextLine();
+    }
+
+    public void getUserInfo() {
+        getLoginAndPassword();
+        System.out.println("Enter name");
+        name = scanner.nextLine();
+        System.out.println("Enter phone number");
+        phoneNumber = scanner.nextLine();
+    }
+
+    @Override
+    public void show() {
+        addOptions();
+        showOptions(options);
+
+        start = true;
```

Comment:
there are no reason to make this variable a global variable. Please change it to local